src/components/FilesList.vue

```diff
@@ -57,13 +90,35 @@ export default {
           label: "modification",
           field: "MODIFIED",
         },
+        {
+          label: "trash_download",
+          field: "",
+        },
       ],
-      entries: [
-      ],
+      entries: [],
     };
   },
-
+  methods: {
+    showButton(entry) {
+      entry.onHover = true;
+    },
+    hideButton(entry) {
+      entry.onHover = false;
+    },
+    download(path) {
+      var msg = { type: "forward", command: "download", path: path };
```

Comment:
Move it away to a service. The view shouldn't know about communication model/api